src/js/components.js

```diff
@@ -49,7 +49,17 @@ export function Word(props, ...children) {
         Char({
           charIndex,
           ...props,
-          onInput: e => charInput(e.target.value, props.wordIndex, charIndex)
+          onInput: e => {
+            const input = e.target.value;
+            charInput(input, props.wordIndex, charIndex);
+
+            if (/[a-zA-Z]/.test(input)) {
+              const nextChildren = this.children[charIndex].$element.nextSibling;
```

Comment:
Maybe we should also allow numbers?

Reply:
Can you provide me a use case where you would allow numbers to be inserted? My thought process was that this was only limited to words - but I could be wrong.